src/frontend/pages/home/Home.js

```diff
@@ -1,11 +1,12 @@
 import "./home.css";
-import { Products, Filter } from "../../components/index";
+import { Products, Filter, Footer } from "../../components/index";
 
 function Home() {
   return (
-    <div className="sidebar-and-products">
+    <div className="filterProductAndFooter">
```

Comment:
footer you provided in the app.js. Then why here?

Reply:
Footer not provided in App.js